set up new broc files

chanceyw opened the pull request.

chanceyw committed:
set up new broc files

chanceyw committed:
added app.js file with generic function

rtablada commented on public/index.html:
```diff
@@ -2,9 +2,12 @@
-
+    <div class="parent">
+    <h1 class="time">Time</h1>
+</div>
```
Run beautify, then remember to include a `script` tag with an `src` attribute pointing to the `app.js` in your project. Remember that just like CSS, nothing is magically loaded in.

chanceyw committed:
added script for js

chanceyw committed:
made new function for current time

chanceyw committed:
set up paddedzeros funtion added progress bar

chanceyw committed:
added console.logs for testing color generator

chanceyw committed:
trying to create hover

rtablada commented on public/app.js:
```diff
@@ -0,0 +1,45 @@
+    return time;
+  }
+
+  var colorSpan = (digit, span) => {
```
Why did you rename this from the `toColorRange` function I sent you?

chanceyw commented:
i was using this to understand it more i have no idea why i renamed it at the time to be honest i changed it back but i am not seeing it what i did wrong.. i guess ill try again in the morning...

chanceyw committed:
changed name of function added some hover elements

chanceyw committed:
renamed right funtion

chanceyw committed:
fixed up the code still getting error for cannot set property of innerhtml null

chanceyw committed:
finished clock fixed hover problem
